go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.18.2
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/utils v0.0.0-20191010214722-8d271d903fe4
```

Comment:
Just for information
It is ok to use: 
https://clearlydefined.io/definitions/git/github/kubernetes/utils/8d271d903fe4c290aa361acfb242cff7bcee96f1